tests/test_md.py

```diff
@@ -532,6 +538,81 @@ def test_multiple_steps_local_consistency(self):
         np.testing.assert_array_equal(summed_pot_xs, xs)
         np.testing.assert_array_equal(summed_pot_boxes, boxes)
 
+    def test_multiple_steps_local_entire_system(self):
+        """Verify that running multiple_steps_local is valid even when consuming the entire system, IE radius ~= inf.
+
+        - Only a single particle, the reference, should not move
+        """
+        mol, _ = get_biphenyl()
+        ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
```

Comment:
nit/curiosity: why not use the default forcefield here?

Reply:
The simple charge forcefield doesn't require openeye, which allows us to test the implementation free of needing a license.

Reply:
Ah, makes sense. I missed that it was not just the older version of the default forcefield.